tests/console/searchindex/create_test.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use phpbb\console\command\searchindex\create;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+require_once __DIR__ . '/base.php';
+require_once __DIR__ . '/mock/search_backend_mock.php';
+
+class create_test extends base
```

Comment:
phpbb_console_searchindex_create_test

Reply:
Same for the other files in tests

Reply:
If I dont remember wrong, phpunit don't find the class name when testing only one file if the class name is different than the file name